cpp/velox/operators/functions/RegistrationAllFunctions.cc

```diff
@@ -37,6 +41,17 @@ void registerCustomFunctions() {
       std::make_unique<RowConstructor>());
 }
 
+// Register some functions that is not accepted by the community, but necessary for gluten.
+void registerFuntionsForGluten() {
```

Comment:
Move to `registerFunctionOverwrite`? It is to override some Velox functions.

Reply:
Thanks. will fix it.